gammapy/stats/variability.py

```diff
@@ -347,18 +358,24 @@ def TimmerKonig_lightcurve_simulator(
             "The power spectrum has to be provided as a callable function."
         )
 
+    if not isinstance(npoints * nchunks, int):
+        raise TypeError("npoints and nchunks must be integers")
```

Comment:
```suggestion
        raise TypeError("The value ``npoints * nchunks`` must be an integer.")
```
you are checking that the multiplication is an integer not that the single variables are

Reply:
The only way that the multiplication is an integer is if both factors are integers, it's just a more compact way of doing the check.